install_config/cluster_metrics.adoc

```diff
@@ -12,307 +12,433 @@ toc::[]
+        - name: PUBLIC_MASTER_URL
+          value: ${PUBLIC_MASTER_URL}
+        - name: MASTER_URL
+          value: ${MASTER_URL}
+        - name: REDEPLOY
+          value: ${REDEPLOY}
+        - name: USE_PERSISTENT_STORAGE
+          value: ${USE_PERSISTENT_STORAGE}
+        - name: HAWKULAR_METRICS_HOSTNAME
+          value: ${HAWKULAR_METRICS_HOSTNAME}
+        - name: CASSANDRA_NODES
+          value: ${CASSANDRA_NODES}
+        - name: CASSANDRA_PV_SIZE
+          value: ${CASSANDRA_PV_SIZE}
+        - name: METRIC_DURATION
+          value: ${METRIC_DURATION}
+    dnsPolicy: ClusterFirst
+    restartPolicy: Never
+    serviceAccount: metrics-deployer
+    volumes:
+    - name: empty
+      emptyDir: {}
+    - name: secret
+      secret:
+        secretName: metrics-deployer
+parameters:
+-
+  description: 'Specify prefix for metrics components; e.g. for "openshift/origin-metrics-deployer:v3.1", set prefix "openshift/origin-"'
+  name: IMAGE_PREFIX
+  value: "openshift/origin-"
+-
+  description: 'Specify version for metrics components; e.g. for "openshift/origin-metrics-deployer:v3.1", set version "v1.1"'
+  name: IMAGE_VERSION
+  value: "latest"
+-
+  description: "Internal URL for the master, for authentication retrieval"
+  name: MASTER_URL
+  value: "https://kubernetes.default.svc:443"
+-
+  description: "External hostname where clients will reach Hawkular Metrics"
+  name: HAWKULAR_METRICS_HOSTNAME
+  required: true
+-
+  description: "If set to true the deployer will try and delete all the existing components before trying to redeploy."
+  name: REDEPLOY
+  value: "false"
+-
+  description: "Set to true for persistent storage, set to false to use non persistent storage"
+  name: USE_PERSISTENT_STORAGE
+  value: "true"
+-
+  description: "The number of Cassandra Nodes to deploy for the initial cluster"
+  name: CASSANDRA_NODES
+  value: "1"
+-
+  description: "The persistent volume size for each of the Cassandra nodes"
+  name: CASSANDRA_PV_SIZE
+  value: "10Gi"
+-
+  description: "How many days metrics should be stored for."
+  name: METRIC_DURATION
+  value: "7"
 ----
 ====
 
-Save it to a file and create it with:
+[[deployer-template-parameters]]
+==== Deployer Template Parameters
 
-----
-$ oc create -f <path/to/heapster-serviceaccount.yaml>
-----
+The deployer template parameter options and their defaults are listed above in the *_metrics.yaml_* file. If required, you can override these values when creating the *metrics deployer*.
 
-Add the *cluster-reader* role to the *Heapster* service account:
+For example, access to the master server is configured by the `*MASTER_URL*`
+parameter. The default value is `https://kubernetes.default.svc:443`, however
+this likely needs to be overridden and configured for your environment. Failing
+to set this parameter properly may result in timeout errors in the deployer
+logs. You may also want to override the value for `*IMAGE_PREFIX*` to suit your
+deployment.
```

Comment:
`IMAGE_PREFIX` is only required if you are building your own images (which is not something that an admin would normally be expected to do, they should be using our existing images). Under normal situations an admin should not have to change this.